codespell_lib/data/dictionary.txt

```diff
@@ -28266,6 +28284,10 @@ trageting->targeting
 tragets->targets
 traiing->trailing, training,
 trailling->trailing
+traingle->triangle
+traingular->triangular
+traingulation->triangulation
+traingulations->triangulations
```

Comment:
Awesome, you could add triangulate(s/d) if you wanted to cover all the bases too.

Reply:
done!